testsuite/features/step_definitions/salt_steps.rb

```diff
@@ -547,6 +547,7 @@
   if verb == 'be' && value == ''
     raise "Output has more than one line: #{output}" unless output.split("\n").length == 1
   elsif verb == 'be'
+    raise "Output value not found : #{output}" unless output.split("\n") > 1
```

Comment:
`'\n'` to make rubocop happy

Reply:
Incorrect, if we use single quotes it will not work.
If rubocop complains, must be due to something else.